code/api/src/migrations/8-updateSubscriptions.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
What does `'use strict'` do?

Reply:
Hey Nicole - https://stackoverflow.com/questions/1335851/what-does-use-strict-do-in-javascript-and-what-is-the-reasoning-behind-it

This resource is what we used to answer that question. During our research into Node.js and how to set up our files, we kept seeing the `'use strict';` syntax at the top of files so we assumed that was something we had to include.